pkg/services/librarypanels/librarypanels_test.go

```diff
@@ -25,6 +25,19 @@ func TestCreateLibraryPanel(t *testing.T) {
 			response = sc.service.createHandler(sc.reqContext, command)
 			require.Equal(t, 400, response.Status())
 		})
+
+	testScenario(t, "When an admin tries to create a library panel that does not exist, it should succeed",
+		func(t *testing.T, sc scenarioContext) {
+			command := getCreateCommand(1, "Text - Library Panel")
+			response := sc.service.createHandler(sc.reqContext, command)
+			require.Equal(t, 200, response.Status())
+
+			var result libraryPanelResult
+			err := json.Unmarshal(response.Body(), &result)
+			l := result.Result.Model["libraryPanel"].(map[string]interface{})
+			require.NoError(t, err)
+			require.Equal(t, l["name"], "Text - Library Panel")
```

Comment:
Can you instead assert the equivalence of `l` itself, e.g. with `cmp.Diff`? Otherwise, there can be unaccounted for fields in it, it also makes the code easier to verify for a reviewer.

Reply:
Yes, I'll fix that!

Reply:
Fixed!